Gemfile

```diff
@@ -45,6 +45,9 @@ group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem 'rubocop', '~> 0.61.1'
+  gem 'rspec-rails', '~> 3.8'
```

Comment:
This gem should be in you `test` group

Reply:
I also thought that gem should be in test group, but rspec-rails documentation suggests we should put it in both development and test group 🤔 
https://github.com/rspec/rspec-rails#installation 

So, should I move that to test group anyway?